build/makefiles/provider.mk

```diff
@@ -102,16 +103,17 @@ $(dest_par): $(bin_target0) Makefile Cargo.toml
 		--compress
 	@echo Created $@
 
-
 # par-full adds all the other targets to the base par
 par-full: $(dest_par) $(bin_targets)
-	# add other defined targets
 	for target in $(par_targets); do \
-	    target_dest=target/$$target/release/$(bin_name);  \
-	    par_arch=`echo -n $$target | sed -E 's/([^-]+)-([^-]+)-([^-]+)(-gnu)?/\1-\3/'`; \
+	    target_dest=target/$${target}/release/$(bin_name);  \
+		if [ $$target = "x86_64-pc-windows-gnu" ]; then \
+			target_dest=$$target_dest.exe;  \
+		fi; \
+	    par_arch=`echo -n $$target | sed -E 's/([^-]+)-([^-]+)-([^-]+)(-gnu.*)?/\1-\3/' | sed 's/darwin/macos/'`; \
```

Comment:
instead of doing the sed here inside this one rule, is there a variable we could change so that target is already correct?

Reply:
Yeah, I might want your help with that as there's a good bit of complexity in the Makefile. Because everything was generally correct other than the target, I wasn't sure if you matched against `x86_64-darwin` or `aarch64-darwin` anywhere else.